<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Venta_carro</class>
 <widget class="QDialog" name="Venta_carro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="rb_tipo_camioneta_venta">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Camioneta</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsb_carro_precio_venta">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_marca_carro_venta">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_estado_bueno_carro_venta">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Bueno</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_cancelar_venta_carro">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio de Venta</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_tipo_pickup_venta">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Pickup</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Marca:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Venta de Carro</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado del Carro</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_estado_reparado_carro_venta">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Reparado</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Placa:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_comprar_venta_carro">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Comprar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_placa_carro_venta">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_estado_malo_carro_venta">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Malo</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_tipo_turismo_venta">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Turismo</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de Carro</string>
   </property>
  </widget>
  <zorder>pb_cancelar_venta_carro</zorder>
  <zorder>rb_tipo_camioneta_venta</zorder>
  <zorder>dsb_carro_precio_venta</zorder>
  <zorder>le_marca_carro_venta</zorder>
  <zorder>rb_estado_bueno_carro_venta</zorder>
  <zorder>label_11</zorder>
  <zorder>rb_tipo_pickup_venta</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>rb_estado_reparado_carro_venta</zorder>
  <zorder>label_3</zorder>
  <zorder>le_placa_carro_venta</zorder>
  <zorder>rb_estado_malo_carro_venta</zorder>
  <zorder>rb_tipo_turismo_venta</zorder>
  <zorder>label_9</zorder>
  <zorder>pb_comprar_venta_carro</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_cancelar_venta_carro</sender>
   <signal>clicked()</signal>
   <receiver>Venta_carro</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
